### PR TITLE
Change the readme to rst format

### PR DESCRIPTION
**Pull request by bobsilverberg**
I needed to make a change to `setup.py` which broke the install (case issue with filename) and I figured I might as well tackle issue #5 at the same time, as they are related. Once merged I will release this as v1.3.1.

@davehunt r?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+   :alt: PyPI
+.. image:: https://img.shields.io/travis/mozilla/fxapom.svg
+   :target: https://travis-ci.org/mozilla/fxapom/
+   :alt: Travis
+.. image:: https://img.shields.io/github/issues-raw/mozilla/fxapom.svg
+   :target: https://github.com/mozilla/fxapom/issues
+   :alt: Issues
+.. image:: https://img.shields.io/requires/github/mozilla/fxapom.svg
+   :target: https://requires.io/github/mozilla/fxapom/requirements/?branch=master
+   :alt: Requirements
```
Comment (davehunt) [inline README.rst]: :heart: I just activated this repository on requires.io

File: README.rst [hunk 2]
```diff
@@ -1,12 +1,25 @@
 Firefox Accounts Page Object Model
 ==================================
-[Selenium WebDriver][webdriver] compatible page object model and utilities for [Firefox Accounts][FxA]
```
Comment (davehunt) [inline README.rst]: Nit: Can we add a blank line after the section header?

File: README.rst [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
+   :target: https://github.com/mozilla/fxapom/issues
+   :alt: Issues
+.. image:: https://img.shields.io/requires/github/mozilla/fxapom.svg
+   :target: https://requires.io/github/mozilla/fxapom/requirements/?branch=master
+   :alt: Requirements
 
 Overview
--------------
+--------
 This package contains a utility to create a test Firefox Account on either the dev or prod instance of Firefox Accounts,
```
Comment (davehunt) [inline README.rst]: Nit: Add a blank line after the section header.

File: README.rst [hunk 4]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
 
 There are two constants available to you to specify the url for either the development environment
 or the production environment, which are:
-- `fxapom.DEV_URL` - the url for the development environment
-- `fxapom.PROD_URL` - the url for the production environment
+* `fxapom.DEV_URL` - the url for the development environment
```
Comment (davehunt) [inline README.rst]: This is not rendering correctly in GitHub, I suspect we need a blank line before the list?

File: README.rst [hunk 5]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
...
+* `fxapom.PROD_URL` - the url for the production environment
```
Comment (davehunt) [inline README.rst]: We need two backquotes to mark this as inline code.

File: README.rst [hunk 6]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
...
-acct = FxATestAccount().create_account()
-```
+
+.. code-block:: python
+
+  from fxapom.fxapom import FxATestAccount
+  acct = FxATestAccount().create_account()
+
 
 Example of creating an account on the development environment, specifying the `DEV_URL`:
```
Comment (davehunt) [inline README.rst]: Two backquotes for inline code.

File: README.rst [hunk 7]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
...
-from fxapom.fxapom import DEV_URL, FxATestAccount
-acct = FxATestAccount(DEV_URL).create_account()
-```
+
+.. code-block:: python
+
+  from fxapom.fxapom import DEV_URL, FxATestAccount
+  acct = FxATestAccount(DEV_URL).create_account()
 
 To sign in via Firefox Accounts, use the `sign_in` method in the `WebDriverFxA` object,
```
Comment (davehunt) [inline README.rst]: Two backquotes for inline code.

File: README.rst [hunk 8]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
...
-Note that we are passing `mozwebqa` into the constructor of `WebDriverFxA`, which is only
-generally available when using our in-house plugin [pytest-mozwebqa][plugin].
+.. code-block:: python
 
-[plugin]: https://github.com/mozilla/pytest-mozwebqa
+  from fxapom.fxapom import WebDriverFxA
+  fxa = WebDriverFxA(mozwebqa)
+  fxa.sign_in(email_address, password)
+
+Note that we are passing `mozwebqa` into the constructor of `WebDriverFxA`, which is only
```
Comment (davehunt) [inline README.rst]: Two backquotes for inline code.

File: README.rst [hunk 9]
```diff
@@ -74,10 +90,14 @@ This software is licensed under the [MPL](http://www.mozilla.org/MPL/2.0/) 2.0:
 Change Log
 ==========
```
Comment (davehunt) [inline README.rst]: Could we make this a subsection (replace `=` with `-`)

File: README.rst [hunk 10]
```diff
@@ -74,10 +90,14 @@ This software is licensed under the [MPL](http://www.mozilla.org/MPL/2.0/) 2.0:
...
 
+1.3.1
+-----
```
Comment (davehunt) [inline README.rst]: These would then become subsubsections (replace `-` with `^`)

File: README.rst [hunk 11]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
...
+
+.. code-block:: python
+
+  from fxapom.fxapom import DEV_URL, FxATestAccount
+  acct = FxATestAccount(DEV_URL).create_account()
 
 To sign in via Firefox Accounts, use the `sign_in` method in the `WebDriverFxA` object,
 passing in the email addresss and password.
 
 Example:
```
Comment (davehunt) [inline README.rst]: I think we can remove this, and just add `:` to the end of the previous sentance.

File: README.rst [hunk 12]
```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
...
+  from fxapom.fxapom import WebDriverFxA
+  fxa = WebDriverFxA(mozwebqa)
+  fxa.sign_in(email_address, password)
+
+Note that we are passing `mozwebqa` into the constructor of `WebDriverFxA`, which is only
+generally available when using our in-house plugin `pytest-mozwebqa <https://github.com/mozilla/pytest-mozwebqa>`_.
 
 To create an account and then use it to sign in, use both tools described above.
 
 Example:
```
Comment (davehunt) [inline README.rst]: I think we can remove this, and just add : to the end of the previous sentance.

File: setup.py [hunk 13]
```diff
@@ -1,12 +1,9 @@
...
-# dependencies
-deps = ['PyFxA==0.0.6']
-
 setup(name='fxapom',
-      version='1.3',
+      version='1.3.1',
       description="Mozilla Firefox Accounts Page Object Model",
-      long_description=open('README.md').read(),
+      long_description=open('README.rst').read(),
       classifiers=[],  # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
```
Comment (davehunt) [inline setup.py]: We should add classifiers :smile: at least 'License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)'

**Comment by bobsilverberg** [comment]
@davehunt This is ready for another review.

File: README.rst [hunk 14]
```diff
@@ -0,0 +1,119 @@
+Firefox Accounts Page Object Model
+==================================
+
+`Selenium WebDriver <http://docs.seleniumhq.org/docs/03_webdriver.jsp>`_ compatible page object model and utilities for `Firefox Accounts <https://accounts.firefox.com>`_
+
+.. image:: https://img.shields.io/pypi/l/fxapom.svg
+   :target: https://github.com/mozilla/fxapom/blob/master/LICENSE
```
Comment (davehunt) [inline README.rst]: Oops, this links to a file that doesn't exist. Let's create a LICENSE file and remove that section from the README.

**Comment by davehunt** [comment]
r+ with the LICENSE comment addressed! Thanks @bobsilverberg for making the README so beautful! :smile:

**Comment by bobsilverberg** [comment]
Thanks @davehunt :)

**Pull request merged**